api/src/test/scala/hmda/api/model/ModelGenerators.scala

```diff
@@ -1,9 +1,9 @@
 package hmda.api.model
 
 import java.util.Calendar
-
 import hmda.model.fi._
 import hmda.model.institution.InstitutionStatus.{ Active, Inactive }
+import hmda.validation.engine.{ Macro, Quality, Syntactical, Validity }
```

Comment:
I don't think this import line is necessary; the whole package is imported on line 8.